modules/4337/CHANGELOG.md

```diff
@@ -2,6 +2,44 @@
 
 This changelog only contains changes starting from version 0.2.0
 
+# Version 0.3.0
+
+## Compiler settings
+
+Solidity compiler: [0.8.23](https://github.com/ethereum/solidity/releases/tag/v0.8.23)
+
+Solidity optimizer: enabled with 10.000.000 runs
+
+## Supported EntryPoint
+
+TBD
+
+## Expected addresses
+
+- `SafeModuleSetup` at `TBD`
+- `Safe4337Module` at `TBD`
+
+## Changes
+
+### Security Fixes
+
+None
+
+### Compatibility Fixes
+
+None
+
+### General
+
+- Use the new `PackedUserOperation` struct from EntryPoint v0.7.0 ((#225)[https://github.com/safe-global/safe-modules/issues/225])
```

Comment:
I think the `[]` and `()` are backwards:
```suggestion
- Use the new `PackedUserOperation` struct from EntryPoint v0.7.0 ([#225](https://github.com/safe-global/safe-modules/issues/225))
```

Reply:
Copilot failed me here :smile:

Reply:
This still needs to be updated right?

Reply:
Oof, I thought I had committed that.